src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -1358,6 +1493,12 @@ private void MergeHidingLookupResults(LookupResult resultHiding, LookupResult re
                             // SPEC: interface type, the base types of T are the base interfaces
                             // SPEC: of T and the class type object. 
 
+                            // PROTOTYPE revisit what kind of type comparison we want when we get to the generic case of extension member lookup
```

Comment:
>revisit what kind of type comparison we want when we get to the generic case of extension member lookup [](http://example.com/codeflow?start=41&length=103)

We probably should be consistent with what ```IsDerivedType``` considers as inheritance between two generic types.